app/Actions/Photo/Create.php

```diff
@@ -126,8 +127,10 @@ public function add(
 			$res = $this->save($this->photo);
 		}
 
-		if ($delete_imported && !is_uploaded_file($this->tmp_name) && ($exists || Configs::get_value('import_via_symlink', '0') !== '1')) {
-			@unlink($this->tmp_name);
+		if ($delete_imported && !is_uploaded_file($this->tmp_name) && ($exists || !$import_via_symlink)) {
```

Comment:
This will conflict with my #888. Whichever goes in first, the other one will need to be rebased...

Reply:
Indeed. Fortunately the conflict is trivial.